CRVFilters/src/CrvCoincidenceClusterFilter_module.cc

```diff
@@ -0,0 +1,152 @@
+//
+// A Filter module aimed to select events using a Likelihood defined with calorimeter cluster info
+//
+// $Id: $
+// $Author: $
+// $Date: $
+//
+// Original author G. Pezzullo
+//
+
+#include "CLHEP/Units/SystemOfUnits.h"
+#include "GlobalConstantsService/inc/GlobalConstantsHandle.hh"
+
+#include "ConfigTools/inc/ConfigFileLookupPolicy.hh"
+
+#include "CalorimeterGeom/inc/Calorimeter.hh"
+#include "CalorimeterGeom/inc/DiskCalorimeter.hh"
+
+#include "GeometryService/inc/GeomHandle.hh"
+#include "GeometryService/inc/GeometryService.hh"
+
+#include "CaloCluster/inc/ClusterMoments.hh"
+
+#include "RecoDataProducts/inc/CrvCoincidenceClusterCollection.hh"
+#include "RecoDataProducts/inc/TrkFitFlag.hh"
+#include "RecoDataProducts/inc/TriggerInfo.hh"
+
+// #include "art/Framework/Core/EDAnalyzer.h"
+#include "art/Framework/Core/EDFilter.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Run.h"
+#include "art/Framework/Principal/Handle.h"
+#include "art/Framework/Principal/Selector.h"
+#include "art/Framework/Principal/Provenance.h"
+
+#include "cetlib_except/exception.h"
+#include "fhiclcpp/ParameterSet.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+#include "canvas/Utilities/InputTag.h"
+
+#include "TDirectory.h"
+#include "TH2F.h"
+#include "TH1D.h"
+#include "TH1F.h"
+#include "TFile.h"
+
+#include <cmath>
+#include <string>
+#include <vector>
+
+
+using namespace std;
+
+namespace mu2e {
+
+
+  class CrvCoincidenceClusterFilter : public art::EDFilter {
+     
+  public:
+    using Name=fhicl::Name;
+    using Comment=fhicl::Comment;
+    struct Config {
+      fhicl::Atom<int> diag{ Name("ddiagLevel"),
+	Comment("Diagnostic Level"), 0};
+      fhicl::Atom<art::InputTag> CRVCCF{ Name("CrvCoincidenceClusterFinder"),
+	Comment("CrvCoincidenceClusterFinder producer")};
+      fhicl::Atom<int> minNCC{ Name("MinNCC"),
+	Comment("Minimum number of cluster of coincidences")};
+      fhicl::Atom<std::string> trgPath{ Name("triggerPath"),
+	Comment("label of the given trigger-path")};
+    };
+
+    enum {
```

Comment:
I am trying this comment again, this time properly tied to the line number.   The enum defined on line 74 of CRVFilters/src/CrvCoincidenceClusterFilter_module.cc  is not used elsewhere in the code.  Please delete it.